server/routes/one-time-code-auth/verifyOneTimeCode.spec.ts

```diff
@@ -0,0 +1,78 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import nock from 'nock'
+import config from '../../config'
+import mockHmppsAuth from '../fixtures/mock-hmpps-auth'
+import assertThat from '../fixtures/supertest-assertions'
+import SuperTestWrapper from '../fixtures/SuperTestWrapper'
+
+describe('Verify One Time Code Integration Tests', () => {
```

Comment:
This is actually a move and update of verifyLink.spec.ts but GitHub went with delete and add for the diff.